python/broadinstitute_cmap/io/GCToo/parse_gctoox.py

```
@@ -1,414 +0,0 @@
-	# open file 
-	gctx_file = h5py.File(full_path, "r", driver = "core")
-
-	# get version
-	my_version = gctx_file.attrs[version_node]
-	if type(my_version) == np.ndarray:
-		my_version = my_version[0]
-
-	# get dsets corresponding to rids, cids 
-	rid_dset = gctx_file[rid_node]
-	cid_dset = gctx_file[cid_node]
-	# store commonly accessed id-related information in dict for O(1) lookup
-	id_info = make_id_info_dict(rid_dset, cid_dset, rid, cid)
-
-	if meta_only == None:
-		# Reads data (or specified slice thereof) from input GCTX to pandas DataFrame
-		data_dset = gctx_file[data_node]
-		data_df = parse_data_df(data_dset, id_info)
-
-		# Reads row metadata (or specified slice thereof) from input GCTX to pandas DataFrame
-		row_group = gctx_file[row_meta_group_node]
-		row_meta_df = make_meta_df("rids", row_group, id_info, convert_neg_666)
-
-		# Reads col metadata (or specified slice thereof) from input GCTX to pandas DataFrame
-		col_group = gctx_file[col_meta_group_node]
-		col_meta_df = make_meta_df("cids", col_group, id_info, convert_neg_666)
-
-		# close file 
-		gctx_file.close()
-
-		# make GCToo instance 
-		curr_GCToo = GCToo.GCToo(src=full_path, version=my_version,
-			row_metadata_df=row_meta_df, col_metadata_df=col_meta_df, data_df=data_df)
-
-		return curr_GCToo
-
-	elif meta_only == "row":
-		# Reads row metadata (or specified slice thereof) from input GCTX to pandas DataFrame
-		row_group = gctx_file[row_meta_group_node]
-		row_meta_df = make_meta_df("rids", row_group, id_info, convert_neg_666)
-
-		# close file 
-		gctx_file.close()
-
-		return row_meta_df
-
-	elif meta_only == "col":
-		# Reads col metadata (or specified slice thereof) from input GCTX to pandas DataFrame
-		col_group = gctx_file[col_meta_group_node]
-		col_meta_df = make_meta_df("cids", col_group, id_info, convert_neg_666)
-
-		# close file 
-		gctx_file.close()
-
-		return col_meta_df	
-
-	else: 
-		logger.error("meta_only argument must be either 'row' or 'col'!")	
```

FYI this should probably throw an exception, not just log the error